index.js

```diff
@@ -0,0 +1,132 @@
+
+const arr = [];
+let id = 2;
```

Comment:
jw. ta nazwa nic nie mówi... `inputId` i pytanie czy Ci czasem nie chodziło o `index` ?
`let id` poza tym wskazuje, że to id się będzie zmieniać w przyszłości. a ID powinno być niezmienne zawsze.